docs/platform/developing/creating-tic.md

```diff
@@ -23,17 +23,57 @@ TIC is an image creator tool used to create images for Tizen. The main features
 2. Run the TIC docker container as a service:
 
    ```shell
+      mkdir -p $HOME/tic-artifacts
       docker login -u [DockerHub_Username] -p [DockerHub_Password]
       docker-compose pull
       docker-compose up -d
    ```
 
 ### Connect to the TIC web-service
 
-Open the web browser and enter `http://127.0.0.1:8088` in the address bar. The TIC web UI screen appears:
+   * Open the web browser and enter `http://127.0.0.1:8088` in the address bar. The TIC web UI screen appears:
 
-  ![TIC web-service](./media/tic-ui.png)
+   ![TIC web-service](./media/tic-ui.png)
 
-Once the Tizen image is created, the final step is to flash the image to a target device for verification. For more information, see [Flashing an Image to RPI](flashing-rpi.md).
+### Creat a new Tizen image and SDK package
+
+#### Settings
+   * Easy
+     * Select predefined set of images
+     * Click `Apply changes` button
+     ![Setting-Easy](./media/tic-ui.png)
+
+   * Advanced
+     * Select `Archtecture`
+     * Import your own `Recipe`, `Repository` and `Package` (optional)
+     * Click `Apply changes` button
+     ![Setting-Advanced](./media/tic-settings-advanced.png)
```

Comment:
This image is missing from the `docs/platform/developing/media` folder and hence doesn't show up in the staging site.